source/hardware/controllers/pioneer_ddj_400.rst

```diff
@@ -0,0 +1,188 @@
+
+Mixer section (p. 8)
+~~~~~~~~~~~~~~~~~~~~
+
+.. figure:: ../../_static/controllers/pioneer_ddj_400_mixer.svg
+   :align: center
+   :width: 65%
+   :figwidth: 100%
+   :alt: Pioneer DDJ-400 (mixer section)
+   :figclass: pretty-figures
+
+   Pioneer DDJ-400 (mixer section)
+
+====  =======================================================  ======================================================================
+No.   Control                                                  Function
+====  =======================================================  ======================================================================
+1     :hwlabel:`TRIM`                                          Adjusts deck prefader gain
+2     :hwlabel:`MASTER`                                        Adjusts master output level
+3     Headphone :hwlabel:`CUE` (Master)                        Sends master output to headphones (adjust with headphone mixing knob)
+4     EQs (:hwlabel:`LOW` :hwlabel:`MID` :hwlabel:`HIGH`)      Boosts or cuts channel frequencies
+5     Channel level indicators                                 Displays channel levels (before being sent through channel faders)
+6     :hwlabel:`HEADPHONES` :hwlabel:`MIXING`                  Adjusts between headphone :hwlabel:`CUE` and :hwlabel:`MASTER` levels sent to headphones
+7     :hwlabel:`FILTER`                                        QuickEffect superknob (by default controls a Filter, but this can be changed in the :ref:`preferences <preferences-equalizers>`)
+8     :hwlabel:`HEADPHONES` :hwlabel:`LEVEL`                   Adjusts output level from :hwlabel:`PHONES` jack
+9     Headphone :hwlabel:`CUE` (Channel)                       Turn on to output channel to headphones
+9     :hwlabel:`SHIFT` + Headphone :hwlabel:`CUE`              Toggles quantize for the current channel
+10    Channel fader                                            Adjusts channel levels output to master
+11    Crossfader                                               Fade between left and right channels
+====  =======================================================  ======================================================================
+
+Effect section (p. 9)
+~~~~~~~~~~~~~~~~~~~~~
+
+.. figure:: ../../_static/controllers/pioneer_ddj_400_effects.svg
+   :align: center
+   :width: 20%
+   :figwidth: 100%
+   :alt: Pioneer DDJ-400 (effect section)
+   :figclass: pretty-figures
+
+   Pioneer DDJ-400 (effect section)
+
+The :hwlabel:`BEAT FX` controls above are mapped to Mixxx's first effect unit as
+follows:
+
+====  =======================================================  ======================================================================
+No.   Control                                                  Function
+====  =======================================================  ======================================================================
+1     :hwlabel:`BEAT` :hwlabel:`<`                             Select effect 1
```

Comment:
```suggestion
1     :hwlabel:`BEAT` :hwlabel:`<`                             Focus effect 1
```
"Focus" is the term we use elsewhere in Mixxx... at least if you're using the `focused_effect` ControlObject.